supplementary_style_guide/glossary_terms_conventions/general_conventions/b.adoc

```diff
@@ -209,6 +305,54 @@ The practice of having both modes together is often referred to as "hybrid", "ag
 
 *See also*: xref:broadcast-n[broadcast (noun)]
 
+// AMQ: General; kept as is
+[discrete]
+[[broker-cluster]]
+==== image:images/yes.png[yes] broker cluster (noun)
+*Description*: A group of brokers to be grouped together in order to share message processing load. In JBoss A-MQ 6, this was called a _network of brokers_.
```

Comment:
Should this start as a full sentence? And do we need the 'in order to' or the 'to be' future/passive bit?:
Maybe this is overkill or incorrect but suggesting:
`A broker cluster is a group of brokers that can share message processing load.`